src/compaction_execute_bench/README.md

```diff
@@ -28,6 +28,9 @@ Set the following environment variables to configure the benchmark:
 
 `VAL_BYTES` : Size of values in bytes (default: 224).
 
+`COMPACTION` : Set to a comma-separated list of SR ids to compact SRs from an existing database instead of
```

Comment:
Nit: Wanna add `COMPRESSION_CODEC` while you're at it? Notice it didn't get included when compression support was added.